### PR TITLE
chore: create stats interface

### PR DESCRIPTION
**Pull request by KalvinVilla**
add stats interface, need to finish ui and refractor the code
Capture d’écran du 2024-06-26 21-27-23

### CODE REVIEW

File: src/components/game-ui/game-interface.vue [hunk 1]
```diff
@@ -71,6 +74,9 @@ function open_spells() {
     .server
       font-size .8em
       color #EEEEEE
+    align-items center
+    > *
+      pointer-events all
```
Comment (Sceat) [inline src/components/game-ui/game-interface.vue:79]: This section is a bit ambiguous, duplicate pointer events ? also please separate the `.server`

File: src/components/game-ui/game-stats.vue [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+  vitality: Vitality
+  chance: Chance
+  health_points: Health
+  soul_points: Soul
+  ap: AP
+  mp: MP
+  points_to_distribute: Points to distribute
+  level: Level
+  experience: Experience
+  increase: +
```
Comment (Sceat) [inline src/components/game-ui/game-stats.vue:16]: a + sign doesn't need to be translated

File: src/components/game-ui/game-stats.vue [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+  experience: Expérience
+  increase: +
+</i18n>
+
+<template lang="pug">
+.game-stats
+  .header {{ t('stats') }}
+  .player-info
+    .avatar
+      img(src="https://assets.aresrpg.world/classe/iop_female.jpg" alt="Character Image")
```
Comment (Sceat) [inline src/components/game-ui/game-stats.vue:39]: this should take the correct skin, you can concatenate `${classe}_${sex}`

**Review by Sceat — COMMENTED** [review]
**Thanks for the feature!** It'll be a bit complex to have it working on chain, but I'll take the PR as a base to make it work 🚀

**Comment by KalvinVilla** [comment]
I imrove  ui,

Need todo :
- [ ] refractor code to improve it
- [ ] add hover to the bars
- [ ] change `+` to svg

If you have some idea for the ui, i will take it


image

**Comment by Sceat** [comment]
it's fine for now, I'll take care of the rest when I have some time and we'll merge!

**Pull request closed without merging**